posthog/models/event_definition.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from posthog.models.team import Team
+from posthog.models.utils import UUIDClassicModel
+
+
+class EventDefinition(UUIDClassicModel):
```

Comment:
You're doing this with event properties too right?

Reply:
Yup, just wanted to do events first to get the heavy lifting out of the way

Reply:
Why `UUIDClassicModel` not `UUIDModel`?